### DISCUSSION TITLE
[Question]: Configuration example needed, 2 streams, different resolutions

### DISCUSSION THREAD

**Discussion by McPaHa** [blakeblackshear/frigate#13101] [Ask A Question]
### What is your question:

Hi!

Don't sure and I think I could manage this with test - error - test work but v. 14.0 how I configure if:

4 cameras with different resolutions each.

Camera 1 main stream 1920x1080 and I want record, snapshots, events etc saved from main stream 
Camera 1 sub stream 704x576 where I want detect

Camera 2 main stream 1920x1080  and I want record, snapshots, events etc saved from main stream 
Camera 2 sub stream 640x360 where I want detect

Camera 3 main stream 1280x720 and I want record, snapshots, events and detect etc saved from main stream 

Camera 4 main stream 1920x1080 and I want record, snapshots, events etc saved from main stream 
Camera 4 sub stream 704x576 where I want detect + detect from audio also

At the moment I have one (main stream) from each cameras where both record and detection happens. Is this wise? What I achieve if I am making detection from sub stream? What if fub stream quality is 640x360? Is that enough for detection?

Sorry about maybe stupid questions...

**Comment by NickM-27** [comment]
if you don't have high CPU usage then I don't think it would make sense to use a sub stream if your sub streams are only 640x360

**Reply by ElectronicBattle** [reply]
> I don't think it would make sense to use a sub stream if your sub streams are only 640x360

I don't understand: we know the [docs](https://docs.frigate.video/frigate/camera_setup#choosing-a-detect-resolution) say that the detection field size is 320x320 and there's no gain for higher resolution larger fields. This is  due to the fact that the frame gets chopped down (unless you're looking for objects which are small in the field of view, in which case a higher res might help).

Is the specific point _here_ the fact that if the CPU burden is low, even when using main stream res,  you might as well enjoy a bit of added detection performance for when there are "small" objects, and it isn't a burden?  In other words, a trade off between CPU burden and data bandwidth vs [ease/likelihood/probability] of detection ?

**Reply by NickM-27** [reply]
Yes, because the resolutions used here are not that high

**Reply by ElectronicBattle** [reply]
got it - yes, if it's 8k it'd be a burden but not if it is ordinary HD.  Thanks Nick.
